### DISCUSSION TITLE
How to clear editor value

### DISCUSSION THREAD

**Discussion by imotality** [sibiraj-s/ngx-editor#376] [Q&A] [answered]
How to clear editor value on reactive forms submit. Like when we have a form and there is this.from.reset() it clears value from it, but not from the editor. editor values are still there. Is there any way to remove it?
tried this.editor.commands.insertHTML('').focus().exec();
and many others with text etc.

**Comment by imotality** [comment]
this.editor.view.dom.innerHTML = '';
found it

**Reply by sibiraj-s** [reply]
See https://github.com/sibiraj-s/ngx-editor/discussions/376#discussioncomment-1916660

**Reply by sibiraj-s** [reply]
Form reset is fixed as well in [v12.1.0](https://github.com/sibiraj-s/ngx-editor/releases/tag/v12.0.1)

**Comment by sibiraj-s** [comment, marked as answer]
https://github.com/sibiraj-s/ngx-editor/discussions/376#discussioncomment-1912946

```js
this.editor.view.dom.innerHTML = ''
```

Though it works, it is not recommended directly modify the view .

If your form is like this.

```ts
class Component {
  form = new FormGroup({
    editor: new FormControl({ value: '', disabled: false }))
  });
}
```

you can do

```ts
this.form.get('editor').reset('')
```

This should work. Not sure why `this.form.reset()` is not working though. I can check.

or directly with editor instance

```js
this.editor.setContent('')
```

## Update

From [v12.0.1](https://github.com/sibiraj-s/ngx-editor/releases/tag/v12.0.1), you can do

```ts
this.form.reset()
```

[a comment by an automated account is not shown]
